packages/fx-core/tests/plugins/resource/bot/unit/generateArmTemplates.test.ts

```diff
@@ -122,7 +122,7 @@ describe("Bot Generates Arm Templates", () => {
     pluginContext.projectSettings!.solutionSettings = azureSolutionSettings;
 
     // Act
-    const result = await botPlugin.generateArmTemplates(pluginContext);
+    const result = await botPlugin.updateArmTemplates(pluginContext);
 
     // Assert
     const provisionModuleFileName = "botProvision.result.bicep";
```

Comment:
Line 170-172 also verify the value of the compiledResult.Provision!.Reference!.xxx instead of just assert existence?

Reply:
Line 169 has wrong indentation, too.

Reply:
Reference value no need to check, the key need to check